test/fixtures/operator/installation.yaml

```diff
@@ -15,5 +15,6 @@ metadata:
 spec:
   channel: stable
   name: snyk-operator
+  installPlanApproval: Automatic
```

Comment:
what's that?

Reply:
Just being explicit about it, but I think it's the default

Basically if an Operator needs to be installed or upgraded, it doesn't need human approval